.github/workflows/verify_kubernetes.yml

```diff
@@ -0,0 +1,42 @@
+name: KubernetesContainerRunner
+on:
+  pull_request:
+    paths:
+      - 'yarn.lock'
+      - '.github/workflows/verify_kubernetes.yml'
+      - 'packages/backend-common/src/util/**'
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+
+    env:
+      CI: true
+      KUBERNETES: true
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: fetch branch master
+        run: git fetch origin master
+
+      - name: use node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: https://registry.npmjs.org/ # Needed for auth
+
+      - name: yarn install
+        uses: backstage/actions/yarn-install@v0.5.3
+        with:
+          cache-prefix: ${{ runner.os }}-v${{ matrix.node-version }}
+
+      - name: bootstrap kind
+        uses: helm/kind-action@v1.3.0
+
+      - name: techdocs-cli kubernetes test
```

Comment:
👀

Reply:
😄 LOL copy paste magic... thanks for spotting this